### ISSUE TITLE
Ledger live not with vite-react-rainbowkit

### ISSUE THREAD

**Issue by omarsy** [wagmi-dev/create-wagmi#49]
Run this template: `npm init wagmi -- --template vite-react-rainbowkit`

Replace wagmi.ts by this: 
```
import { connectorsForWallets, getDefaultWallets } from '@rainbow-me/rainbowkit'
import { configureChains, createClient } from 'wagmi'
import { goerli, mainnet } from 'wagmi/chains'
import { publicProvider } from 'wagmi/providers/public'
import {
  argentWallet,
  trustWallet,
  ledgerWallet,
} from '@rainbow-me/rainbowkit/wallets';
const { chains, provider, webSocketProvider } = configureChains(
  [mainnet, ...(import.meta.env?.MODE === 'development' ? [goerli] : [])],
  [
    publicProvider(),
  ],
)

const { wallets } = getDefaultWallets({
  appName: 'My wagmi + RainbowKit App',
  chains,
})
const connectors = connectorsForWallets([
  ...wallets,
  {
    groupName: 'Other',
    wallets: [
      argentWallet({ chains }),
      trustWallet({ chains }),
      ledgerWallet({ chains }),
    ],
  },
]);

export const client = createClient({
  autoConnect: true,
  connectors,
  provider,
  webSocketProvider,
})

export { chains }

```
Now  try ledge live  We will have an issue

**Comment by jxom** [comment]
What is the issue? 

Can you please provide a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) via a CodeSandbox or repository?

Will reopen when done!

**Comment by omarsy** [comment]
I give you the step to reproduce it @jxom . I don't know if you want anything else. Just add ledger live in your vite-react-rainbowkit you will see the ledger live will not work. But if you want I can create repository but I think it will be more difficult to debug for us

**Comment by omarsy** [comment]
I see the issue only for vite

**Comment by omarsy** [comment]
On my side in your example. I cannot connect with ledge too 
<img width="799" alt="image" src="https://user-images.githubusercontent.com/15034695/216000345-94e9e968-bbf8-4430-8ca1-e59efc03f5aa.png">
after I have this
<img width="710" alt="image" src="https://user-images.githubusercontent.com/15034695/216000458-aa24b3f8-0d8f-4f2b-acf4-45f07d9215f4.png">
we have this when the connection is not success

instead of having this
<img width="582" alt="image" src="https://user-images.githubusercontent.com/15034695/216001533-eaeae9eb-4140-40d6-b6f2-eaa230947db5.png">

**Comment by omarsy** [comment]
I try to debug on my side it seems the URI that we give is not a good URI
When it fail: ledgerlive://wc?uri=wc%3A%401%3Fbridge%3Dhttps%253A%252F%252Fd.bridge.walletconnect.org%26key%3D
When it is success: ledgerlive://wc?uri=wc%3A563b3545-39df-46f9-a35f-9df700477f4d%401%3Fbridge%3Dhttps%253A%252F%252Ft.bridge.walletconnect.org%26key%3D52a7ac26e42c256dbd24cb16ec162f02ea38a4edb087279fcc9d3b5d8060d2ce

**Comment by jxom** [comment]
I don't believe this is an issue with create-wagmi, but rather downstream in RainbowKit or Ledger.

AFAIK, there is a PR to update the Ledger SDK in RainbowKit.

https://github.com/rainbow-me/rainbowkit/pull/938

Might be worth keeping an eye on that PR.

cc/ @nickbytes @hlopes-ledger

**Comment by hlopes-ledger** [comment]
Hi @omarsy!

There is currently a bug in Ledger Live in which you have to close the Wallet Connect screen by clicking the X at the top right or choosing another screen from the left side bar. You are shown the Wallet Connect screen each time you disconnect, so you will have to do that each time.

**Comment by omarsy** [comment]
@hlopes-ledger I did it then after that I have the issue with the wrong uri `ledgerlive://wc?uri=wc%3A%401%3Fbridge%3Dhttps%253A%252F%252Fd.bridge.walletconnect.org%26key%3D`
<img width="806" alt="image" src="https://user-images.githubusercontent.com/15034695/216014742-80a3e778-1272-4faf-ac49-90b3a4104f4c.png">

**Comment by omarsy** [comment]
But the issue seems only happen on Vite I test it with wepback I didn't have an issue

**Comment by hlopes-ledger** [comment]
I can't reproduce it with the code on your repository. I always get the Accept/Decline screen if I close the Wallet Connect one before connecting again.

**Comment by omarsy** [comment]
![screen-capture (2)](https://user-images.githubusercontent.com/15034695/216025053-e9865981-9e5d-4559-b314-34ebe0f03a40.gif)

**Comment by omarsy** [comment]
Yes the second time after the fist connect fail if you don't refresh the page it will work.
But it doesnt fix my issue because even if the second time the connection is success. I cannot sign the message, I have this error
<img width="339" alt="image" src="https://user-images.githubusercontent.com/15034695/216025860-b7c9055c-8348-4d42-ab56-e4bc5f36d27f.png">
But when I switch to webpack everything is working fine

**Comment by omarsy** [comment]
I create the issue here. I find why have the issue only on vite 
https://github.com/wagmi-dev/references/issues/99
